### ISSUE TITLE
Can I create applications in Python (pygtk)?

### ISSUE THREAD

**Issue by itsMichael** [elementary/houston#444]
How can I distribute applications, builded in python?

**Comment by btkostner** [comment]
You can distribute a python application, but we would highly recommend using vala as all of our docs and ecosystem are based around it.

Three major things you need to make sure:

- The project is buildable with `debuild`. We have no docs about this because it's in Python, but it shouldn't be too hard to find information about it with a quick Google.

- The project conforms to our HIG. It should still be a native gtk3 application. The user should know no difference this app and a native vala app.

- The project includes icons, a desktop file, and appstream file. We are currently only accepting regular desktop apps, and these files are needed for desktop integration. 

tl;dr: as long as it's buildable with `debuild` and has everything a normal elementary app would have.
